src/pages/Library.tsx

```diff
@@ -1,6 +1,21 @@
+import { useLoaderData } from "react-router-dom";
 import { Container } from "@mui/material";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import CardMedia from "@mui/material/CardMedia";
+import Typography from "@mui/material/Typography";
+import { CardActionArea } from "@mui/material";
+import { useNavigate } from "react-router-dom";
+import HeadPhonesImage from "../assets/images/headphones.jpg";
 
 export default function LibraryPage() {
+  const lists = useLoaderData() as SpotifyApi.PlaylistObjectSimplified[];
+  const navigate = useNavigate();
+
+  function handleOnClick(id: string) {
```

Comment:
The `on` in the name is not necessary. The function handles something. In this case a click on an item or playlist.

I'd call it `handleItemClick` or `handlePlaylistClick`.

Reply:
thanks